File: JavaLab/src/main/java/ch2/section2/Quiz1.java
package ch2.section2;

public class Quiz1 {
    public static void main(String[] args) {
        int month = 11;

        if(month == 3){
            System.out.println("봄입니다.");
        }else if(month == 4){
            System.out.println("봄입니다.");
        }else if(month == 5){
            System.out.println("봄입니다.");
        }else if(month == 6){
            System.out.println("여름입니다.");
        }else if(month == 7){
            System.out.println("여름입니다.");
        }else if(month == 8){
            System.out.println("여름입니다.");
        }else if(month == 9){
            System.out.println("가을입니다.");
        }else if(month == 10){
            System.out.println("가을입니다.");
        }else if(month == 11){
            System.out.println("가을입니다.");
        }else if(month == 12){
            System.out.println("겨울입니다.");
        }else if(month == 1){
            System.out.println("겨울입니다.");
        }else if(month == 2){
            System.out.println("겨울입니다.");
        }else {
            System.out.println("1~12의 숫자가 아닙니다.");
        }

        if(month == 3 || month == 4 || month == 5){
            System.out.println("봄입니다.");
        }else if(month == 6 || month == 7 || month == 8){
            System.out.println("여름입니다.");
        }else if(month == 9 || month == 10 || month == 11){
            System.out.println("가을입니다.");
        }else if(month == 12 || month == 1 || month == 2){
            System.out.println("겨울입니다.");
        }else {
            System.out.println("1~12의 숫자가 아닙니다.");
        }
    }
}

File: KotlinLab/KotlinLab/src/main/kotlin/ch2/section4_control/Quiz1.kt
package org.example.ch2.section4_control

class Quiz1 {
//    fun calcBonus(salary: Int, bonusRate: Double): String{
//        //step1.................
////        if(salary <= 0 || bonusRate <= 0){
////            return "잘못된 데이터를 입력했습니다."
////        }else {
////            val bonus = (salary * bonusRate).toInt()
////            return "지급받을 보너스는 $bonus 원 입니다."
////        }
//
//
//        //step2.................
////        val result = if(salary <= 0 || bonusRate <= 0){
////            "잘못된 데이터를 입력했습니다."
////        }else {
////            val bonus = (salary * bonusRate).toInt()
////            "지급받을 보너스는 $bonus 원 입니다."
////        }
////        return result
//
//        //step3......................
////        return if(salary <= 0 || bonusRate <= 0){
////            "잘못된 데이터를 입력했습니다."
////        }else {
////            val bonus = (salary * bonusRate).toInt()
////            "지급받을 보너스는 $bonus 원 입니다."
////        }
//    }

    fun calcBonus(salary: Int, bonusRate: Double) = if(salary <= 0 || bonusRate <= 0){
        "잘못된 데이터를 입력했습니다."
    }else {
        val bonus = (salary * bonusRate).toInt()
        "지급받을 보너스는 $bonus 원 입니다."
    }
}

fun main() {
    val obj = Quiz1()
    println(obj.calcBonus(1000, 0.25))
}